rpcs3/Emu/RSX/rsx_decode.h

```diff
@@ -10,7 +10,12 @@
 
 namespace rsx
 {
-	std::string print_boolean(bool b);
+	enum class true_or_false_printing : u8 {};
```

Comment:
what a weird name xD
Just call it boolean_to_string or something like that

Reply:
Fixed.